cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -16,7 +16,7 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
     var CourseOutlineXBlockModal, SettingsXBlockModal, PublishXBlockModal, HighlightsXBlockModal,
         AbstractEditor, BaseDateEditor,
         ReleaseDateEditor, DueDateEditor, GradingEditor, PublishEditor, AbstractVisibilityEditor,
-        StaffLockEditor, UnitAccessEditor, ContentVisibilityEditor, TimedExaminationPreferenceEditor,
+        StaffLockEditor, UnitAccessEditor, DiscussionEditor, ContentVisibilityEditor, TimedExaminationPreferenceEditor,
```

Comment:
Like the above comments, I think we can buck the cargo-cult trend here, remove this declaration (so we don't have to keep changing existing lines), and then just declare the variable below.